project/plugins.sbt

```diff
@@ -19,7 +19,7 @@ addSbtPlugin("com.jsuereth" % "sbt-pgp" % "2.0.1")
 
 addSbtPlugin("de.heikoseeberger" % "sbt-header" % "5.6.0")
 
-addSbtPlugin("org.scala-js" % "sbt-scalajs" % "0.6.33")
+addSbtPlugin("org.scala-js" % "sbt-scalajs" % "1.1.0")
```

Comment:
> "It is not forward binary compatible with 1.0.x: libraries compiled with 1.1.0 cannot be used with 1.0.x."

https://users.scala-lang.org/t/announcing-scala-js-1-1-0/6053/3?u=sjrd

So it would be better to stay on 1.0.1 here and be compatible with both Scala.js 1.0 and 1.1.